src/components/TimelineEvent.js

```diff
@@ -1,10 +1,24 @@
 import React from 'react';
 import './TimelineEvent.css';
 import Timestamp from './Timestamp';
+import PropTypes from 'prop-types';
 
-const TimelineEvent = () => {
-  
-  return;
-}
+const TimelineEvent = (props) => {  
+  return (
+    <ul className="TimelineEvent">
+        <li>person: {props.person}</li>
+        <li>status: {props.status}</li>
+        <li>timeStamp: <Timestamp time={props.timeStamp} /></li>
+    </ul>
+  )
+
+  TimelineEvent.propTypes = {
+    timelineevent: PropTypes.shape({
+      person: PropTypes.string.isRequired,
+      status: PropTypes.string,
+      timeStamp: PropTypes.instanceOf(Date)
+    })
+  }
+};
```

Comment:
You put prop types inside the function.

```suggestion
  }
  TimelineEvent.propTypes = {
    timelineevent: PropTypes.shape({
      person: PropTypes.string.isRequired,
      status: PropTypes.string,
      timeStamp: PropTypes.instanceOf(Date)
    })
  };
```